### PR TITLE
[IMPORTED] Reference Links and some code examples added

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) updates reference links to Plugins and adds some examples to the existing topics.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/layouts/xml-manage.html

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -80,8 +80,10 @@ You can use either the `<link src="js/sample.js"/>` or the `<script src="js/samp
 
 The path to assets is specified relatively to one the following locations:
 
--  `<theme_dir>/web`-
--  `<theme_dir>/<Namespace>_<Module>/web`-
+-  CSS and LESS files are stored under `<theme_dir>/web/css/`
+-  Fonts related files are stored under `<theme_dir>/<Namespace>_<Module>/web/fonts/`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:84]:
```suggestion
-  Font related files are stored in the `<theme_dir>/<Namespace>_<Module>/web/fonts/` directory
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 2]
```diff
@@ -80,8 +80,10 @@ You can use either the `<link src="js/sample.js"/>` or the `<script src="js/samp
...
+-  Static assets like images are stored under `<theme_dir>/<Namespace>_<Module>/web/images/`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:85]:
```suggestion
-  Static assets, like images, are stored in the `<theme_dir>/<Namespace>_<Module>/web/images/` directory
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 3]
```diff
@@ -80,8 +80,10 @@ You can use either the `<link src="js/sample.js"/>` or the `<script src="js/samp
 
 The path to assets is specified relatively to one the following locations:
 
--  `<theme_dir>/web`-
--  `<theme_dir>/<Namespace>_<Module>/web`-
+-  CSS and LESS files are stored under `<theme_dir>/web/css/`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:83]:
```suggestion
-  CSS and LESS files are stored in the  `<theme_dir>/web/css/` directory.
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 4]
```diff
@@ -80,8 +80,10 @@ You can use either the `<link src="js/sample.js"/>` or the `<script src="js/samp
...
+-  Fonts related files are stored under `<theme_dir>/<Namespace>_<Module>/web/fonts/`
+-  Static assets like images are stored under `<theme_dir>/<Namespace>_<Module>/web/images/`
+-  JS files are stored under `<theme_dir>/<Namespace>_<Module>/web/js/`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:86]:
```suggestion
-  JS files are stored in the `<theme_dir>/<Namespace>_<Module>/web/js/` directory
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 5]
```diff
@@ -609,6 +617,8 @@ class AddBodyClassToProductPagePlugin
 
 As result, the `<body>` tag has a new `my-new-body-class` class on all product pages.
 
+For more information about the Plugins refer to the [Plugins in Magento]({{ page.baseurl }}/extension-dev-guide/plugins.html)
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:620]:
```suggestion
For more information about Plugins refer to [Plugins in Magento]({{ page.baseurl }}/extension-dev-guide/plugins.html)
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just some small changes

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 6]
```diff
@@ -80,8 +80,10 @@ You can use either the `<link src="js/sample.js"/>` or the `<script src="js/samp
 
 The path to assets is specified relatively to one the following locations:
 
--  `<theme_dir>/web`-
--  `<theme_dir>/<Namespace>_<Module>/web`-
+-  CSS and LESS files are stored in the  `<theme_dir>/web/css/` directory.
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:83]:
```suggestion
-  CSS and LESS files are stored in the `<theme_dir>/web/css/` directory.
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 7]
```diff
@@ -609,6 +617,8 @@ class AddBodyClassToProductPagePlugin
 
 As result, the `<body>` tag has a new `my-new-body-class` class on all product pages.
 
+For more information about Plugins refer to [Plugins in Magento]({{ page.baseurl }}/extension-dev-guide/plugins.html)
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:620]:
```suggestion
For more information about Plugins, refer to [Plugins in Magento]({{ page.baseurl }}/extension-dev-guide/plugins.html)
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Two missed corrections - PR will be approved after these - apologies

**Comment by ajithkumar-maragathavel** [comment]
@BarnyShergold Thanks for the review. I have updated the PR.

**Review by BarnyShergold — APPROVED** [review]
Cheers!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8761`.

**Pull request merged**